src/inc/Request.php

```diff
@@ -192,12 +192,11 @@ public function setRouteParams(array $routeParams)
      */
     public function getParameter($param, $default = '')
     {
-        $value = $default;
         if (array_key_exists($param, $this->parameters)) {
-            $value = $this->parameters[$param];
+            return $this->parameters[$param];
```

Comment:
Which one is the "happy path"? I'd personally say when there's a parameter to be retrieved. So I'd put it the other way round and say 

```php
if (! array_key_exists($param, $this->parameters)) {
    return $default;
}

return $this->parameters[$param];
```

Reply:
That would be better. I just didn't want to make too many changes in this PR. :) 6411dd7